vivarium/core/process.py

```diff
@@ -155,9 +156,15 @@ class can provide a ``defaults`` class variable to specify the
                 '_emit': True,
                 '_updater': 'set'}))
 
-        self.parameters.setdefault('time_step', DEFAULT_TIME_STEP)
+        self._set_timestep()
+
         self.schema: Optional[dict] = None
 
+    def _set_timestep(self) -> None:
+        self.parameters.setdefault('timestep', DEFAULT_TIME_STEP)
+        if self.parameters.get('time_step'):
+            self.parameters['timestep'] = self.parameters['time_step']
```

Comment:
Should we delete `time_step` from the parameters?
* Pros: We can be confident there isn't any code left relying on reading `time_step`
* Cons: If any process is trying to read `self.parameters['time_step']`, they'll break. This is technically a breaking change

Reply:
@U8NWXD I'm inclined to leave it, despite it being kind of clunky. I am personally never certain if it is "time step" or "timestep", and am pretty sure people use both spelling. So this clunky solution at least would avoid that ambiguity.